README.md

```diff
@@ -1 +1,14 @@
+# Testing
 
+Testing can be run either through the launch script, or manually.
+To use the launch script,run the following command in the root folder:
+
+```bash
+. scripts/launch_frontend.sh
```

Comment:
there shouldn't really be a frontend to launch, as the goal is to have it be static. we don't need any other scripts than build and maybe test and those can both be defined in `package.json`